test/parallel/test-child-process-spawn-typeerror.js

```diff
@@ -36,7 +36,16 @@ assert.doesNotThrow(function() {
 // verify that invalid argument combinations throw
 assert.throws(function() {
   spawn();
-}, /Bad argument/);
+}, /TypeError: "file" argument must be a string/);
```

Comment:
In the other test, the regex is declared once and used multiple times, here it's declared multiple times. Perhaps simplify? e.g.

```
const err = /TypeError: "file" argument must be a string/;
assert.throws(() => spawn(), err);
assert.throws(() => spawn(''), err);
assert.throws(() => {
  const file = { toString() { throw new Error('foo'); } };
  spawn(file);
}, err);
```